rust/toolchain.bzl

```diff
@@ -6,6 +6,27 @@ load(":utils.bzl", "relative_path")
 
 ZIP_PATH = "/usr/bin/zip"
 
+def _get_rustc_env(ctx):
+  version = ctx.attr.version if hasattr(ctx.attr, "version") else "0.0.0"
+  v1, v2, v3 = version.split(".")
+  if "-" in v3:
+    v3, pre = v3.split("-")
+  else:
+    pre = ""
+  return [
+    "CARGO=bazel",  # Bazel is actually performing the build, not cargo
```

Comment:
I'm not sure if this is appropriate. This envvar is used occasionally inside of build scripts to invoke cargo itself. Those folks would be in for a rude awakening when trying to shell out to `cargo` and getting `bazel` instead..

Reply:
Since it is for build step not running the build.rs script it shouldn't matter, but since I don't need it I removed it.